src/screens/MapList.tsx

```diff
@@ -15,7 +15,8 @@ export default function MapList() {
       },
     })
       .then((response) => response.json())
-      .then((data) => setLocations(data["connecticut"]));
+      .then((data) => setLocations(data["connecticut"]))
+      .catch((err) => console.log(err));
```

Comment:
A good idea, but out of scope for this PR. Please move to a separate error handling/craftsmanship PR.